.github/ISSUE_TEMPLATE/improve-existing-docs.yml

```diff
@@ -0,0 +1,64 @@
+name: Improve existing docs
+about: Make a suggestion to improve our existing documentation.
+title: ''
```

Comment:
We should be able to leave out the `title` attribute if blank is the default we want for the title.

```suggestion
```